content/api/commands/mount.md

```diff
@@ -0,0 +1,607 @@
+other libraries as well.
+
+### Vuetify
+
+This example shows how to set a global `cy.mount` command that configures
+[Vuetify](https://vuetifyjs.com/), ensuring components that utilize the UI
+library display properly during test runs.
+
+Vuetify is a plugin that must be registered and have a few custom attributes on
+the root element setup for styling to appear correct.
+
+<code-group-vue2-vue3>
+<template #vue2>
+
+```js
+import { mount } from '@cypress/vue'
+import vuetify from '../../src/plugins/vuetify'
+
+Cypress.Commands.overwrite('mount', (comp, options = {}) => {
+  // Add attributes needed for Vuetify styling
+  const root = document.getElementById('__cy_root')
+  if (!root.classList.contains('v-application')) {
+    root.classList.add('v-application')
+  }
+  root.setAttribute('data-app', 'true')
+
+  // vuetify import calls Vue.use(Vuetify) directly
+  // so no need to call it directly here
+
+  return mount(comp, {
+    vuetify,
+    ...options,
+  })
+})
+```
+
+</template>
+<template #vue3>
+
+```js
+import { mount } from '@cypress/vue'
+import vuetify from '../../src/plugins/vuetify'
+
+Cypress.Commands.overwrite('mount', (comp, options = {}) => {
+  // Setup options object
+  options.global = options.global || {}
+  options.global.plugins = options.global.plugins || []
+
+  // Add attributes needed for Vuetify styling
+  const root = document.getElementById('__cy_root')
+  if (!root.classList.contains('v-application')) {
+    root.classList.add('v-application')
+  }
+  root.setAttribute('data-app', 'true')
+
+  // Add Vuetify plugin
+  options.global.plugins.push({
+    install(app) {
+      app.use(vuetify)
+    },
+  })
+
+  return mount(comp, options)
+})
+```
+
+</template>
+</code-group-vue2-vue3>
+
+Typings:
+
+```ts
+import { mount } from '@cypress/vue'
+
+type MountParams = Parameters<typeof mount>
+type OptionsParam = MountParams[1]
+
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      /**
+       * Helper mount function for Vue Components
+       * @param component Vue Component or JSX Element to mount
+       * @param options Options passed to Vue Test Utils
+       */
+      mount(component: any, options?: OptionsParam): Chainable<any>
+    }
+  }
+}
+```
+
+Usage:
+
+```ts
+import Button from './Button.vue'
+
+it('Shows a button', () => {
+  cy.mount(Button, {
+    slots: {
+      default: () => 'Click Me',
+    },
+  })
+  cy.contains('Click Me').should('exist')
+})
+```
+
+### Vue Router
+
+To wire up a plugin such as Vue Router, you can create a custom command to
+register the plugin and pass in a custom implementation of the router via the
+options param.
+
+<code-group-vue2-vue3>
+<template #vue2>
+
+```js
+import { mount } from '@cypress/vue'
+import Vue from 'vue'
+import VueRouter from 'vue-router'
+import { router } from '../../src/router'
+
+Cypress.Commands.add('mountWithRouter', (comp, options = {}) => {
+  // Add the VueRouter plugin
+  Vue.use(VueRouter)
```

Comment:
In the Vue Test Utils [Using with Vue Router](https://vue-test-utils.vuejs.org/guides/using-with-vue-router.html#using-with-vue-router) guide, they do `const localVue = createLocalVue()`. Is this something we should be doing as well?

Reply:
we don't expose a createLocalVue function but I'll bring it up to the CT team and see if we should